app/uk/gov/hmrc/apiplatformmicroservice/apidefinition/services/ExtendedApiDefinitionForCollaboratorFetcher.scala

```diff
@@ -36,7 +36,7 @@ class ExtendedApiDefinitionForCollaboratorFetcher @Inject() (
     subscriptionsForCollaboratorFetcher: SubscriptionsForCollaboratorFetcher
   )(implicit ec: ExecutionContext) {
 
-  def fetch(serviceName: String, developerId: Option[DeveloperIdentifier])(implicit hc: HeaderCarrier): Future[Option[ExtendedApiDefinition]] = {
+  def fetch(serviceName: String, developerId: Option[UserId])(implicit hc: HeaderCarrier): Future[Option[ExtendedApiDefinition]] = {
```

Comment:
should param names change to match the type? (same applies in many other places)

Reply:
Yes.